apps/api/src/config/env.validators.ts

```diff
@@ -56,6 +57,8 @@ export const envValidators = {
       STRIPE_API_KEY: str({ default: undefined }),
       STRIPE_CONNECT_SECRET: str({ default: undefined }),
       NOVU_INTERNAL_SECRET_KEY: str({ default: '' }),
+      KEYLESS_ORGANIZATION_ID: str({ desc: 'Required organizationId for Keyless authentication' }),
+      KEYLESS_USER_EMAIL: str({ desc: 'Required email for Keyless authentication' }),
```

Comment:
I would saly let's add here default ''? To match with the rest of the keys here?

Reply:
hmmm but then that would mean that the API could bootstrap in an invalid state, although the EU does not need those variables..

Reply:
Yep, let's add a default here. Also, did you already created the org in prod and staging? we can't merge this PR before that